Layer/Pdo/Pdo.php

```diff
@@ -194,7 +194,12 @@ public function lastInsertId($name = null)
     public function prepare($statement, array $options = [])
     {
         if (!isset($options[\PDO::ATTR_CURSOR])) {
-            $options[\PDO::ATTR_CURSOR] = \PDO::CURSOR_SCROLL;
+            try {
+                $this->getConnection()->getAttribute(\PDO::ATTR_CURSOR);
+                $options[\PDO::ATTR_CURSOR] = \PDO::CURSOR_SCROLL;
```

Comment:
Also, if this patch works, then this is a bug in PDO with SQLite driver and we should open a bug on bugs.php.net, do you agree?